lfs.c

```diff
@@ -411,6 +411,17 @@ static inline void lfs_superblock_tole32(lfs_superblock_t *superblock) {
     superblock->attr_max    = lfs_tole32(superblock->attr_max);
 }
 
+static inline bool is_node_not_in_list(struct lfs_mlist *head,
```

Comment:
nit: Prefix with `lfs_`. Thoughts on `lfs_mlist_isopen` for consistency with the other functions?

Reply:
done, changed to `lfs_mlist_isused`